battle.h

```diff
@@ -80,6 +81,8 @@ class Battle : public AListElem
 		void WriteSides(ARegion *,Unit *,Unit *,AList *,AList *,int,
 				ARegionList *pRegs );
 
+		// void WriteBattleStats(ArmyStats *);
```

Comment:
need to delete?

Reply:
yes, looks like that

Reply:
i don't see any usage of this function did u intend to add it but did not add?